tests/publiccloud/run_ltp.pm

```diff
@@ -141,19 +156,47 @@ sub run {
 
     assert_script_run($log_start_cmd);
 
-    my $cmd = 'perl -I runltp-ng runltp-ng/runltp-ng ';
-    $cmd .= '--logname=ltp_log --verbose ';
-    $cmd .= '--timeout=1200 ';
-    $cmd .= '--run ' . get_required_var('LTP_COMMAND_FILE') . ' ';
-    $cmd .= '--exclude \'' . get_var('LTP_COMMAND_EXCLUDE') . '\' ' if get_var('LTP_COMMAND_EXCLUDE');
-    $cmd .= '--backend=ssh';
-    $cmd .= ':user=' . $instance->username;
-    $cmd .= ':key_file=' . $instance->ssh_key;
-    $cmd .= ':host=' . $instance->public_ip;
-    $cmd .= ':reset_command=\'' . $reset_cmd . '\'';
-    $cmd .= ':ssh_opts=\'-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no\' ';
-    $cmd .= '--json_filter=openqa ';
-    assert_script_run($cmd, timeout => get_var('LTP_TIMEOUT', 30 * 60));
+    my $ltp_timeout = get_var('LTP_TIMEOUT', 30 * 60);
```

Comment:
this variable is used in only [one place ](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15583/files#diff-49293c308cb7b8e8f6bbfc7f14a6fa90507cd690aa9d5ab41c1f60f96e4cb5ecR199), I think we can drop this variable and just directly call `get_var('LTP_TIMEOUT', 30 * 60)` in place where variable used now